apiv2/views.py

```diff
@@ -1349,8 +1348,8 @@ def create_apiv2_key(request):
             api_client.accepted_tos = form.cleaned_data['accepted_tos']
             api_client.save()
             form = ApiV2ClientForm()
-            logger.info('%s <> (ApiV2 Auth:%s Dev:%s User:%s Client:%s)' %
-                        ('new_credential', None, request.user.username, None, api_client.client_id))
+            api_logger.info('%s <> (ApiV2 Auth:%s Dev:%s User:%s Client:%s)' %
+                            ('new_credential', None, request.user.username, None, api_client.client_id))
```

Comment:
small thing, but the first string interpolation probably isn't needed - can just put 'new_credential' directly in there, right?

Reply:
absolutely!